lib/pages/settings_dashboard/settings_profile/settings_profile_view_mobile.dart

```diff
@@ -119,27 +151,45 @@ class SettingsProfileViewMobile extends StatelessWidget {
               Positioned(
                 bottom: SettingsProfileViewMobileStyle.positionedBottomSize,
                 right: SettingsProfileViewMobileStyle.positionedRightSize,
-                child: InkWell(
-                  onTap: onAvatarTap,
-                  child: Container(
-                    decoration: BoxDecoration(
-                      color: Theme.of(context).colorScheme.primary,
-                      borderRadius: BorderRadius.circular(
-                        SettingsProfileViewMobileStyle.avatarSize,
-                      ),
-                      border: Border.all(
-                        color: Theme.of(context).colorScheme.onPrimary,
-                        width:
-                            SettingsProfileViewMobileStyle.iconEditBorderWidth,
+                child: MenuAnchor(
+                  controller: menuController,
+                  builder: (
+                    BuildContext context,
+                    MenuController menuController,
+                    Widget? child,
+                  ) {
+                    return GestureDetector(
+                      onTap: () {
+                        if (PlatformInfos.isWeb) {
```

Comment:
why would you need this since it's used for mobile ?

Reply:
it's for web responsive